Source/Swig/typemap.c

```diff
@@ -1967,8 +1967,9 @@ static void replace_embedded_typemap(String *s, ParmList *parm_sublist, Wrapper
 	    String *parm = Getitem(l, i);
 	    char *eq = strchr(Char(parm), '=');
 	    char *c = Char(parm);
-	    if (eq && (eq - c > 0)) {
-	      String *name = NewStringWithSize(c, (int)(eq - c));
+	    int size = (int)(eq - c);
+	    if (eq && size > 0) {
+	      String *name = NewStringWithSize(c, size);
```

Comment:
I'm not seeing the problem, can you elaborate?
size is now computed with an "undefined" eq, but it's not a crash, and eq is checked for null before comparing size and protects all of the remaining code in the block.